Sources/ArcGISToolkit/Documentation.docc/Tutorials/JobManagerTutorial.tutorial

```diff
@@ -0,0 +1,138 @@
+@Tutorial(time: 30) {
+    @Intro(title: "JobManager Tutorial") {
+        The `JobManager` manages saving and loading jobs so that they can continue to run if the app is backgrounded or even terminated.
+        
+        @Image(source: JobManagerTutorial, alt: "An image of the JobManager component.")
+    }
+    
+    @Section(title: "Getting started with the JobManager") {
+        @ContentAndMedia {
+            @Image(source: JobManagerTutorial, alt: "An image of the JobManager component.")
+            In this tutorial you will use a job that generates an offline map. The job will connect to a service, ask the service to generate an offline map and then download the map.
+        }
+        
+        @Steps {
+            @Step {
+                Create the JobManagerTutorialView struct. Add properties and start the UI for adding a job to the job manager. 
+                
+                Begin by adding a JobManager property as an observable object. Add state properties for `job`, `error`, and a boolean that you'll use to show progress when adding a job. Start the UI by adding a button to take a map offline and a progress indicator to show that the job is being added. This will be the UI before a job has been added (job == nil).
```

Comment:
```suggestion
                Begin by adding a JobManager property as an observable object. Add state properties for `job`, `error`, and a boolean that you'll use to show progress when adding a job. Start the UI by adding a button to take a map offline and a progress indicator to show that the job is being added. This will be the UI before a job has been added (`job == nil`).
```